src/applications/static-pages/facilities/FacilityListWidget.jsx

```diff
@@ -0,0 +1,156 @@
+      const bName = b.attributes.name;
+      if (aName < bName) {
+        return -1;
+      }
+
+      if (aName > bName) {
+        return 1;
+      }
+
+      return 0;
+    });
+
+    return sortedFacilities.map(facility => {
+      const location = {
+        attributes: facility.attributes,
+      };
+      let address = buildAddressArray(location);
+
+      if (address.length !== 0) {
+        address = address.join(', ');
+      } else {
+        // If we don't have an address fallback on coords
+        const { lat, long } = location.attributes;
+        address = `${lat},${long}`;
+      }
+      return (
+        <div
+          key={facility.id}
+          className="usa-grid vads-u-background-color--gray-lightest vads-u-margin-bottom--2p5 vads-u-padding-y--1p5"
+        >
+          <section className="usa-width-one-half">
+            <h3 className="vads-u-margin-bottom--2p5">
+              <a
+                href={facility.attributes.website}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {facility.attributes.name}
+              </a>
+            </h3>
+            <address className="vads-u-margin-bottom--1p5">
+              <div>{facility.attributes.address.physical.address1}</div>
+              <div>
+                {facility.attributes.address.physical.city},{' '}
+                {facility.attributes.address.physical.state}{' '}
+                {facility.attributes.address.physical.zip}
+              </div>
+            </address>
+            <div className="vads-u-margin-bottom--1p5">
+              <a
+                href={`https://maps.google.com?saddr=Current+Location&daddr=${address}`}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Directions
+              </a>
+            </div>
+            <div className="vads-u-margin-bottom--1p5">
+              <div className="main-phone">
+                <strong>Main phone: </strong>
+                <a
+                  href={`tel:${facility.attributes.phone.main.replace(
+                    /[ ]?x/,
+                    '',
+                  )}`}
+                >
+                  {facility.attributes.phone.main.replace(/[ ]?x/, '')}
+                </a>
+              </div>
+              <div className="mental-health-clinic-phone">
+                <strong>Mental health clinic: </strong>
+                <a
+                  href={`tel:${facility.attributes.phone.mentalHealthClinic.replace(
+                    /[ ]?x/,
+                    '',
+                  )}`}
+                >
+                  {facility.attributes.phone.mentalHealthClinic.replace(
+                    /[ ]?x/,
+                    '',
+                  )}
+                </a>
+              </div>
+            </div>
+            <div className="location-details-link">
+              <a href="#" className="usa-button usa-button-secondary">
```

Comment:
Is this going to have a link eventually?

Reply:
yes, once we get the facility details pages